src/Data/Record/Unsafe.js

```diff
@@ -24,3 +24,7 @@ exports.unsafeDeleteFn = function(label, rec) {
   }
   return copy;
 };
+
+exports.unsafeHasFn = function(label, rec) {
+  return {}.hasOwnProperty.call(rec, label);
```

Comment:
How about using `Object.prototype.hasOwnProperty.call(rec, label)` so that a new object isn't created?

Reply:
It's using the same code the compiler generates in record updates. Engines will optimize this out.